posts/month-in-mathlib-aug-2021.md

```diff
@@ -0,0 +1,76 @@
+---
+author: 'Mathlib community'
+category: 'month-in-mathlib'
+date: 2021-08-31 08:56:51+00:00
+description: ''
+has_math: true
+link: ''
+slug: month-in-mathlib-aug-2021
+tags: ''
+title: This month in mathlib (Aug 2021)
+type: text
+---
+
+# This month in mathlib (Aug 2021)
```

Comment:
```suggestion
This post summarizes some of the activity that happened in mathlib in August.
```
The post title is part of the template. We don't want to repeat it, and it's good to have some space between the title and the first h2.

Reply:
aah, good point. That means we should also update the bash script that generated that line.

Reply:
Except that it wasn't added by the script, but by me manually :facepalm: